.vsts-ci.yml

```diff
@@ -1,5 +1,5 @@
 # g++
-# Build your c++ projects with g++.
+# Build your C++ projects with g++ and GNU Make.
 # https://aka.ms/yaml
 
 queue: 'Hosted Linux'
```

Comment:
I had to rename this to 'Hosted Linux Preview' to queue the build.

Reply:
Thank you for improving this!  Feel free to change it to "Hosted Linux Preview", but I'm happy to quickly commit that directly to master after you merge this PR.